src/extensions/default/brackets-show-whitespace/main.js

```diff
@@ -44,8 +44,8 @@ define(function (require, exports, module) {
     
     // --- Settings ---
     
-    var commandID          = "denniskehrig.ShowWhitespace.toggle";
-    var preferencesID      = "denniskehrig.ShowWhitespace";
+    var commandID          = "allowWhiteSpace.toggle";
```

Comment:
@humph does this already require turning off whatever "denniskehrig.ShowWhitespace" is in some other part of the code? (extension loader or some other place?)